functions/src/utils/firebase.js

```diff
@@ -1,5 +1,23 @@
 const functions = require('firebase-functions');
 const admin = require('firebase-admin');
-admin.initializeApp();
+const firebase = require('firebase');
+const config = require('../config/index');
+const serviceAccount = require('./firebaseAdmin');
+const firebaseConfig = {
+    apiKey: config.firebaseConfig.apiKey,
+    appId: config.firebaseConfig.appId,
+    authDomain: config.firebaseConfig.authDomain,
+    databaseURL: config.firebaseConfig.databaseURL,
+    measurementId: config.firebaseConfig.measurementId,
+    messagingSenderId: config.firebaseConfig.messagingSenderId,
+    projectId: config.firebaseConfig.projectId,
+    storageBucket: config.firebaseConfig.storageBucket,
+};
```

Comment:
Could you destruct all the config.firebaseConfig, I think it is quicker to read that way